lib/common/zstd_internal.h

```diff
@@ -285,9 +285,9 @@ typedef enum {
 *  Private declarations
 *********************************************/
 typedef struct seqDef_s {
-    U32 offset;         /* offset == rawOffset + ZSTD_REP_NUM, or equivalently, offCode + 1 */
+    U32 offset;    /* offset == rawOffset + ZSTD_REP_NUM, or equivalently, offCode + 1 */
     U16 litLength;
-    U16 matchLength;
+    U16 mlBase;    /* mlBase == matchLength - MINMATCH */
```

Comment:
Would a more natural name be `mlCode`, by analogy to `offCode`?

Reply:
`mlCode` is already taken,
it relates to the FSE code actually entropy-encoded.
`mlBase` is the starting value that leads to later transformation `mlBase => mlCode + mlBits`.